hack/e2e-test.sh

```diff
@@ -79,7 +79,7 @@ function cleanup {
             mkdir -p "$ARTIFACTS"
         fi
     uninstall_kjobctl
```

Comment:
Actually looks like  `uninstall_kjobctl` should move to right, no?

Reply:
You right. Thanks!